python/lsst/ts/ataos/ataos_csc.py

```diff
@@ -491,6 +622,8 @@ async def do_enableCorrection(self, id_data):
         """
         self.assert_enabled('enableCorrection')
         self.assert_any_corrections(id_data)
+
+        # FIXME: What does this do? Do we still need them everywhere?
```

Comment:
are you talking about the sleep(0.)? in this case, yes, we want this here.. this will allow the CSC a chance to execute other tasks while executing this command.

Reply:
Then why does do_enableCorrection have two `asyncio.sleep(0)` calls?

Reply:
.. and do_disableCorrection

Reply:
To give control back twice to the event loop.

Reply:
doesn't it only give it back for 0 seconds ??